StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  UpdatePaymentMethodViewModel.swift
+//  StripePaymentSheet
+//
+//  Created by Joyce Qin on 11/15/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+import UIKit
+
+class UpdatePaymentMethodViewModel {
+    let paymentMethod: STPPaymentMethod
+    static let supportedPaymentMethods: [STPPaymentMethodType] = [.card, .USBankAccount, .SEPADebit]
```

Comment:
Nit: On the ordering here, let's put all static variables up on their own and not in between more "normal" variables for readability. 

```swift
class UpdatePaymentMethodViewModel {
    static let supportedPaymentMethods: [STPPaymentMethodType] = [.card, .USBankAccount, .SEPADebit]


    let paymentMethod: STPPaymentMethod
}

Reply:
It's used in VerticalSavedPaymentMethodsViewController and SavedPaymentMethodCollectionView when checking if a payment method should allow editint